### DISCUSSION TITLE
Slip transmission m'

### DISCUSSION THREAD

**Discussion by KorayKuzkaya** [mtex-toolbox/mtex#1700] [Ask Anything] [answered]
Dear mtex community,

on this website it says that the misorientation factor m' goes from -1 to 1.

https://stabix.readthedocs.io/en/latest/slip_transmission.html

In mtex, the value m' goes from 0 to 1. Is that the same value and if so, why does it only go from 0 to 1?

**Comment by Tijmenvermeij** [comment, marked as answer]
In Mtex the absolute value is taken, see the mprime function in Mtex.

On the website you refer to, the mprime values in the examples are negative because the slip directions are basically opposite. For crystallographic slip, this is not really logical since dislocations can also move in opposite directions... However, I'm not 100% sure if this fully excludes negative mprime values. 

In any case, I've never seen people use negative mprime values.
